ros_ign_gazebo_demos/launch/air_pressure.launch.py

```diff
@@ -36,7 +36,8 @@ def generate_launch_description():
     bridge = Node(
         package='ros_ign_bridge',
         executable='parameter_bridge',
-        arguments=["/air_pressure@sensor_msgs/msg/FluidPressure@ignition.msgs.FluidPressure"],
+        arguments=['/air_pressure@sensor_msgs/msg/FluidPressure@ignition.msgs.FluidPressure'],
+        parameters=[{'qos_overrides./air_pressure.publisher.reliability' : 'best_effort'}],
```

Comment:
While it is good to have a demo of how someone can override a QoS setting, I don't think we want to publish 'best_effort' reliability for all sensor data. Note, we decided to make the QoS for gazebo_plugins default to 'reliable' for a better out-of-box experience.

Reply:
Oh thanks for bringing this up, I just saw https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1224. Interesting, I guess [rclcpp::SensorDataQoS](https://docs.ros2.org/foxy/api/rclcpp/classrclcpp_1_1SensorDataQoS.html) can be misleading. Not sure if people have considered splitting it between `SensorDataPublisherQoS` and `SensorDataSubscriberQoS`.